### PR TITLE
Add WARP Routes OORT

### PR DESCRIPTION
**Pull request by novacypher5**

### CODE REVIEW

File: deployments/warp_routes/OORT/bsc-oort-config.yaml
Comment (paulbalaji) [inline deployments/warp_routes/OORT/bsc-oort-config.yaml:1]: rename this file to `bsc-oortmainnet-config.yaml`

also, why do you have two routes? one for bsc<>eth<>oort, the other just bsc<>oort?

**Comment by novacypher5** [comment]
changeset added, linted and prettier looks good

Screenshot from 2024-12-02 16-20-08

File: deployments/warp_routes/OORT/logo.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="480" height="480" fill="none"><path stroke="#BAC4D7" stroke-linecap="round" stroke-linejoin="round" stroke-width="49.091" d="M252.101 91.657a148.59 148.59 0 0 0-149.342 90.349 148.584 148.584 0 0 0 39.596 169.995 148.583 148.583 0 0 0 173.894 15.06 148.582 148.582 0 0 0 68.232-160.657"/><path fill="#0058FF" d="M349.659 76.363c-18.966 0-34.341 15.375-34.341 34.341s15.375 34.341 34.341 34.341S384 129.67 384 110.704s-15.375-34.34-34.341-34.34Z"/></svg>
```
Comment (paulbalaji) [inline deployments/warp_routes/OORT/logo.svg:1]: sorry one more change, also ran it through an svg cropper https://svgcrop.com/
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="66.54 66.59 346.27 346.27"><path stroke="#BAC4D7" stroke-linecap="round" stroke-linejoin="round" stroke-width="49.091" d="M252.101 91.657a148.59 148.59 0 0 0-149.342 90.349 148.584 148.584 0 0 0 39.596 169.995 148.583 148.583 0 0 0 173.894 15.06 148.582 148.582 0 0 0 68.232-160.657"></path><path fill="#0058FF" d="M349.659 76.363c-18.966 0-34.341 15.375-34.341 34.341s15.375 34.341 34.341 34.341S384 129.67 384 110.704s-15.375-34.34-34.341-34.34Z"></path></svg>
```

**Review by paulbalaji — COMMENTED** [review]
lgtm as soon as png is updated!

**Review by paulbalaji — APPROVED** [review]

**Pull request merged**